packages/opencensus-core/src/metrics/export/types.ts

```diff
@@ -135,16 +135,8 @@ export interface TimeSeries {
   readonly points: Point[];
 }
 
-/** Properties of a LabelValue. */
-export interface LabelValue {
-  /** The value for the label */
-  readonly value: string;
-  /**
-   * If false the value field is ignored and considered not set.
-   * This is used to differentiate a missing label from an empty string.
-   */
-  readonly hasValue: boolean;
-}
+/** The LabelValue type. null value indicates an unset. */
+export type LabelValue = string|null;
```

Comment:
Is `|null` necessary? It looks like this project has `strictNullChecks` set to `false` in `tsconfig.json`

Reply:
We have plans to enable it in the future